avaframe/com3Hybrid/com3Hybrid.py

```diff
@@ -41,8 +44,16 @@ def maincom3Hybrid(cfgMain, cfgHybrid):
     """
     avalancheDir = cfgMain['MAIN']['avalancheDir']
     demOri = getInput.readDEM(avalancheDir)
-    # get comDFA configuration path for hybrid model
-    hybridModelDFACfg = pathlib.Path('com3Hybrid', 'hybridModel_com1DFACfg.ini')
+
+    # setup outputs folder
+    oPath = pathlib.Path(avalancheDir, 'Outputs', 'com3Hybrid')
+    fU.makeADir(oPath)
+
+    # get comDFA configuration and save to file
+    com1DFACfg = cfgUtils.getModuleConfig(com1DFA, fileOverride='', modInfo=False, toPrint=False,
+        onlyDefault=cfgHybrid['com1DFA_override']['defaultConfig'])
+    com1DFACfg, cfgHybrid = cfgHandling.applyCfgOverride(com1DFACfg, cfgHybrid, com1DFA, addModValues=True)
+    com1DFACfgFile = cfgUtils.writeCfgFile(avalancheDir, com1DFA, com1DFACfg, fileName='com1DFA_settings', filePath=oPath)
```

Comment:
could we maybe put this guy in Work to avoid any confusion in the end. Otherwise there is a com1DFA_settings.ini and the com3HybridCfg.ini in the output folder. Problem is that the  com1DFA_settings.ini  is updated multiple times wherase the com3HybridCfg.ini is the one that should be used to reproduce the results.

Reply:
I moved it to Work for the com1DFACfg.ini